src/main/resources/data-repository-openapi.yaml

```diff
@@ -1961,6 +1961,8 @@ definitions:
         $ref: '#/definitions/DatePartitionOptionsModel'
       intPartitionOptions:
         $ref: '#/definitions/IntPartitionOptionsModel'
+      rowCount:
```

Comment:
Maybe not for this PR, but it is looking to me like time to split the snapshot table from the dataset table.